CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.  
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).  
 
+## [Next version - 2.2.0]
+
+### Added
+- Public: Added support for Spanish language. The SDK can now be used in Spanish by adding `language: {locale: 'es'}` to the initialization options. If `language` is not present or the wrong locale tag is provided, the locale will default to `en`.
```

Comment:
why the nesting on `language: {locale: 'es'}`?

Reply:
It's nested because before `language` was accepting more options apart from `locale`. I agree that doesn't need to be nested anymore so I can change it to just using `locale`

Reply:
Got it. I think ` language: 'es'` is simple enough. It can also be extended to customisation, if we allow it to accept either a string or an object.